api/handlers_movers.go

```diff
@@ -213,3 +213,98 @@ func (s *server) MoverShowHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write(j)
 }
+func (s *server) MoverShowrunHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := vars["account"]
+	group := vars["group"]
+	name := vars["name"]
+
+	orch, err := s.newDatasyncOrchestrator(
+		r.Context(),
+		account,
+		&sessionParams{
+			role: fmt.Sprintf("arn:aws:iam::%s:role/%s", account, s.session.RoleName),
+			policyArns: []string{
+				"arn:aws:iam::aws:policy/AWSDataSyncReadOnlyAccess",
+				"arn:aws:iam::aws:policy/ResourceGroupsandTagEditorReadOnlyAccess",
+			},
+		},
+	)
+
+	if err != nil {
+		handleError(w, errors.Wrap(err, "unable to create datasync orchestrator"))
+		return
+	}
+
+	//Got info about the move
+	respL, errL := orch.datamoverDescribe(r.Context(), group, name)
+	if errL != nil {
+		handleError(w, errL)
+		return
+	}
+	// from move get info of all executions (runs) for a taskArn (moveId)
+	resp, err := orch.taskRunsFromTaskArn(r.Context(), *respL.Task.TaskArn)
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+
+	j, err := json.Marshal(resp)
+	if err != nil {
+		handleError(w, apierror.New(apierror.ErrInternalError, "failed to marshal json", err))
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(j)
+}
+func (s *server) MoverShowrunbyIDHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
add a newline before this and rename to `RunShowHandler`

Reply:
Done